textext/requirements_check.py

```diff
@@ -624,19 +624,35 @@ def find_tkinter(self):
 
         return RequirementCheckResult(True, ["TkInter is found"])
 
+    def _find_inkscape_without_inkex(self):
+        """
+        Returns either (stdout_line, executable) or None
+        If the first is returned, stdout_line is the output of
+        inkscape --version and executable is the path to inkscape
+        """
+        try:
+            executable = self.find_executable('inkscape')['path']
+            stdout, stderr = defaults.call_command([executable, "--version"])
+            stdout_line = stdout.decode("utf-8", 'ignore')
+            return stdout_line, executable
+        except (KeyError, OSError):
+            return None
+
     def find_inkscape_1_4(self):
         try:
-            # When we call this from Inkscape we need this call
             import inkex.command as iec
-            stdout_line = iec.inkscape("", version=True)
-            executable = iec.which("inkscape")
         except ImportError:
+            result = self._find_inkscape_without_inkex()
+        else:
             try:
-                executable = self.find_executable('inkscape')['path']
-                stdout, stderr = defaults.call_command([executable, "--version"])
-                stdout_line = stdout.decode("utf-8", 'ignore')
-            except (KeyError, OSError):
-                return RequirementCheckResult(False, ["inkscape is not found"])
+                result = iec.inkscape("", version=True), iec.which("inkscape")
```

Comment:
Under Windows, we have a problem here under special circumstances. While `import inkex.command` succeeds `iec.inkscape` may fail if `inkscape.exe` is not in the path. In that case we need to fall back to `_find_inkscape_without_inkex` again. I fixed this in the old approach by commit 58f846b (`CommandError` derives from `IOError`). 

We need to make this smooth without repeated try / except blocks.

Reply:
I suppose that approach works too.